core/controllers/editor.py

```diff
@@ -42,6 +45,8 @@
 from core.domain import value_generators_domain
 from core.domain import visualization_registry
 from core.platform import models
+from google.appengine.api import app_identity
```

Comment:
Don't import directly from google.appengine.api in here -- can you use the stuff in core.platform.app_identity instead? You can declare new functions there if you need them.